### PR TITLE
shop.driveplaza.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/112316

<details><summary>Screenshot:</summary>

Снимок экрана 2022-03-23 в 21 41 17

Снимок экрана 2022-03-23 в 21 41 38

Снимок экрана 2022-03-23 в 21 40 59

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -39,6 +39,7 @@ financial-field.com,spaia.jp,yomiuri.co.jp,fujinkoron.jp,mikle.jp,onayamifree.co
 !
 !
 cosme.net##.atcosme-HeaderAdContent
+shop.driveplaza.com###sideBnrArea
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:42]: These are self-promo. If banners link to their own sites, suspect self-promo.

File: MobileFilter/sections/specific_web.txt [hunk 2]
```diff
@@ -32,6 +32,7 @@ scmp.com,theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalo
 cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,siirtpress.com,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
 !
 stolica-s.su##.ad-fullscreen
+shop.driveplaza.com###topBnrArea
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:35]: I don't see this element, where you found?

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/specific_web.txt (hunk 2 above)
Comment (jellizaveta) [reply MobileFilter/sections/specific_web.txt:35]: On mobile screen

<details><summary>Screenshot:</summary>

Снимок экрана 2022-03-28 в 12 14 45

</details><br/>

Comment (Yuki2718) [reply MobileFilter/sections/specific_web.txt]: Found thanks, they look to be self-promo.

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**